### DISCUSSION TITLE
土耳其调兵

### DISCUSSION THREAD

**Discussion by KongBai000-0** [share121/inter-knot#3057] [其他]
怎么评
![Uploading 屏幕截图 2024-08-19 152756.png…]()

**Comment by warning-Kirov-reporting** [comment]
<img src="https://bbs-static.miyoushe.com/static/2024/03/21/cfc12c6552ad36ddf88bd651e90da441_8544373733813917056.png" alt="丽都漫步-苍角" width="75" >你的冰没调出来

**Comment by KongBai000-0** [comment]
难崩，我图片呢！！！！！！！
